tests/settings/ConfigurationGatheringTest.py

```diff
@@ -354,28 +333,25 @@ class ConfigurationGatheringCollectionTest(unittest.TestCase):
 
     def setUp(self):
         self.old_argv = sys.argv
-        self.log_printer = LogPrinter(NullPrinter())
 
     def tearDown(self):
-        close_objects(self.log_printer)
+        close_objects()
         sys.argv = self.old_argv
 
     def test_get_filtered_bears(self):
         sys.argv = ['coala', '-I']
 
         with bear_test_module():
-            local_bears, global_bears = get_filtered_bears(
-                None, self.log_printer)
+            local_bears, global_bears = get_filtered_bears(None)
 
         self.assertEqual(len(local_bears['cli']), 13)
 
         with bear_test_module():
-            local_bears, global_bears = get_filtered_bears(
-                ['Java'], self.log_printer)
+            local_bears, global_bears = get_filtered_bears(['Java'])
 
         self.assertEqual(len(local_bears['cli']), 2)
         self.assertEqual(str(local_bears['cli'][0]),
-                         "<class 'JavaTestBear.JavaTestBear'>")
-        self.assertEqual(str(local_bears['cli'][1]),
                          "<class 'LineCountTestBear.LineCountTestBear'>")
+        self.assertEqual(str(local_bears['cli'][1]),
+                         "<class 'JavaTestBear.JavaTestBear'>")
```

Comment:
This is latest change. I don't what changed in last few commits. But these two are now coming in reverse order in test output. I don't understand why are we getting list over here, instead of set ?
@Makman2 @sils